package.json

```diff
@@ -16,15 +16,14 @@
     "lint": "xo generator.js",
     "generate-example": "npm run example-generate",
     "example-generate": "rm -rf tram-one-example && node generator.js tram-one-example",
-    "postexample-generate": "npm --prefix ./tram-one-example install",
-    "example-build": "npm --prefix ./tram-one-example run build",
-    "example-start": "npm --prefix ./tram-one-example start",
```

Comment:
removing prefix, not sure why it wasn't working, but `cd`-ing into the directory and running the command seems to be the accepted approach for this.

Reply:
also, `postexample-generate` was 100% unnecessary, since the `npm install` happens as the last part of generating the project.